guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

```diff
@@ -1,10 +1,6 @@
 [id="Enabling_the_Client_Repository_{context}"]
 = Enabling the {project-client-name} Repository
 
-ifdef::foreman-el,katello[]
-You require the Katello plug-in to complete this procedure.
-endif::[]
-
```

Comment:
This should probably be kept. It wasn't removed in the previous PR.

Reply:
It's actually something that can be done in master too.  The whole file is only included on `ifdef::satellite[]` so it was never reached.